README.md

```diff
@@ -42,7 +42,7 @@ This command will create an `ApplicationDecorator` file.
 
 Generate a new decorator with the `decorator` generator:
 
-    $ rails generate decorator user
+    $ rails generate dekorator:decorator user
```

Comment:
I'm not a fan of this long syntax... In Komponent we use `rails generate component foobar`, not `rails generate komponent:component foobar`. There is no other way?

Reply:
Yes I know, it's why it was like this at first.

Pundit, ActionPolicy do it with gem name as namespace. I searched but didn't found a way to do some alias. And didn't found any gem who do like we do in komponent :/

Reply:
Draper does: https://github.com/drapergem/draper#generators

Reply:
I did a second generator that invoke the first one. Don't know if it's a good think.